components/application_manager/utils.h

```diff
@@ -126,9 +126,8 @@ class Utils {
 
     class Timeout {
```

Comment:
What's the reason for the implementation being in the header? Unless there's some templating involved it should usually go in the .cpp file.

Reply:
This is not a refactoring PR.

Reply:
Good point.